browser_history/cli.py

```diff
@@ -50,10 +60,18 @@ def main():
     It parses arguments from sys.argv and performs the appropriate actions.
     """
     args = parser.parse_args()
+    assert args.type in ['history','bookmarks','all'], \
+                        f"Type should be one of all, {AVAILABLE_TYPES}"
 
+    h_outputs = b_outputs =None
+    fetch_map = {'history':[h_outputs ,get_history(),'history'+args.format],
+                 'bookmarks':[b_outputs,get_bookmarks(),'bookmarks'+args.format]}
```

Comment:
This is a nice idea, but compromises code readability a lot.

For example, let's say I was reading line `105` - I can't figure out what the `[2]` denotes there without scrolling up to this part (which I should also find) and then going back there. This would not be a good experience for new contributors.

An alternative would be making this a `dict` (this would also not be very readable - it will look like ["history"]["filename"]) or an entirely new class (maybe a [dataclass](https://docs.python.org/3/library/dataclasses.html)).

Let me know if you have any other ideas for this.

Reply:
I removed this completely, since bookmarks and history extraction barely used the same functions anymore , I think its best if the user selects one at a time else it makes the cli function very shabby

Reply:
That's a good decision xD